src/userpage/UserPage.js

```diff
@@ -1,26 +1,126 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Input from '../components/Input';
 import Header from '../components/Header';
 import ContentContainer from '../components/ContentContainer';
+import ToDo from '../todopage/ToDo';
+import Entry from '../newspage/Entry';
+import styled from 'styled-components';
+import SubmitButton from '../components/SubmitButton';
+import ColorInput from '../components/ColorInput';
 
-export default function UserPage({ user, onUserChange }) {
-  function onSubmit(event, onUserChange) {
+const uid = require('uid');
+
+const UserBox = styled.span`
+  background: ${props => props.color};
+  border-radius: 5px;
+  display: inline-block;
+  list-style: none;
+  margin: 5px;
+  padding: 5px;
+  box-shadow: 0 0 5px 1px rgba(221, 221, 221, 1);
+
+  :last-child {
+    background: skyblue;
+    color: white;
+    padding: 5px 10px;
+    font-weight: bold;
+  }
+`;
+
+const CreateUserForm = styled.form``;
```

Comment:
you have several forms where you don't use styles. i am not sure if it is necessary to have a special named component without any styles. maybe think about using the normal html element instead of a non styled component